llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp

```diff
@@ -11175,6 +11179,9 @@ bool BoUpSLP::isLoadCombineCandidate(ArrayRef<Value *> Stores) const {
 }
 
 bool BoUpSLP::isTreeTinyAndNotFullyVectorizable(bool ForReduction) const {
```

Comment:
I think ideally we'd only increment the counter and force return `true` in the cases where this function would return `false`, but ultimately it doesn't matter too much, it just makes the counter ranges larger